osf/models/node.py

```diff
@@ -370,6 +370,19 @@ class AbstractNode(DirtyFieldsMixin, TypedModel, AddonModelMixin, IdentifierMixi
     custom_storage_usage_limit_public = models.DecimalField(decimal_places=9, max_digits=100, null=True, blank=True)
     custom_storage_usage_limit_private = models.DecimalField(decimal_places=9, max_digits=100, null=True, blank=True)
 
+    all_responses = GenericRelation('osf.SchemaResponses')
+
+    @property
+    def schema_responses(self):
```

Comment:
this is what the `all_responses` GenericRelation does. The fact that that wasn't clear makes me wonder: should we rename that? `all_schema_responses`, maybe? Is confusing because there's two levels of plurality :/

Reply:
Well maybe just schema responses then? It's going to appear as `schema_reponses.all()` anyway right?

Reply:
legit, I'll make that change in the other PR.

Any other name changes we want to explore now that we've started writing code around these models?

Reply:
The whole revision wording, I would call everything schema_responses, but I just followed the API contract. I would change it, but lets check with the FE.

Reply:
Also `justification` on the model should be `revision_justification`  to match the API contract.